<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hourlyPaid</class>
 <widget class="QDialog" name="hourlyPaid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnaddhourlyPaid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txthourlyPaidSocialSecuritynumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblhourlyPaidTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Register new hourly paid employee&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancelhourlyPaid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txthourlyPaidDoneHours">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblhourlyPaidsocialsalary">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Done Hours&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txthourlyPaidName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblhourlyPaidName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Name&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(FirstName LastName)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblhourlyPaidsocialSecuritynumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Social security &lt;br/&gt;number&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(DDMMYY-123A)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txthourlyPaidhcompensation">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblhourlyPaidsocialsalary_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Hourly&lt;br/&gt;Compensation&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txthourlyPaidName</tabstop>
  <tabstop>txthourlyPaidSocialSecuritynumber</tabstop>
  <tabstop>txthourlyPaidDoneHours</tabstop>
  <tabstop>txthourlyPaidhcompensation</tabstop>
  <tabstop>btnaddhourlyPaid</tabstop>
  <tabstop>btncancelhourlyPaid</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
